schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -345,7 +354,10 @@ public static void repairAll(
   }
 
   /**
-   * Repair namespaces and tables defined in the schema file that are in an unknown state.
+   * Repair namespaces and tables defined in the schema that are in an unknown state, such as the
+   * namespace or table exists in the underlying storage but not its ScalarDB metadata or vice
+   * versa. This will recreate the namespaces, the tables, the secondary indexes and their metadata
+   * if necessary.
```

Comment:
```suggestion
   * Repair namespaces and tables defined in the schema that are in an unknown state, such as the
   * namespace or table exists in the underlying storage but not its ScalarDB metadata or vice
   * versa. This will re-create the namespaces, the tables, the secondary indexes, and their metadata
   * if necessary.
```